startup_http/handler.go

```diff
@@ -42,10 +41,13 @@ func WriteJSON(ctx context.Context, w http.ResponseWriter, status int, value int
 	}
 	w.WriteHeader(status)
 
-	err := json.MarshalWrite(w, value, json.FormatNilSliceAsNull(false), json.FormatNilMapAsNull(false))
+	body, err := json.Marshal(value)
```

Comment:
This breaks the semantic we had previously. We want to switch to json/v2 as soon as it is available.